src/resources/version.ts

```diff
@@ -129,10 +129,11 @@ class VersionResource extends CrudResource<typeof SVersion['schema'], ModelKey,
     return newProject;
   }
 
-  public async getPrototype(id: VersionID) {
+  public async getPrototype(id: VersionID, body: { sanitizePassword?: boolean } = {}) {
     this._assertModelID(id);
 
-    const { data } = await this.fetch.get<VersionPrototype>(`${this._getCRUDEndpoint(id)}/prototype`);
+    const query = body.sanitizePassword ? `?sanitizePassword=${body.sanitizePassword}` : '';
+    const { data } = await this.fetch.get<VersionPrototype>(`${this._getCRUDEndpoint(id)}/prototype${query}`);
```

Comment:
Prototype Share View needs this to sanitize the password, before `creator-api` sends a response. We want to avoid exposing the password over the wire, especially since the prototype share link might be used in a public space.